manifests/m/Microsoft/EdgeBeta/93.0.961.37/Microsoft.EdgeBeta.installer.yaml

```diff
@@ -0,0 +1,36 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: Microsoft.EdgeBeta
+PackageVersion: 93.0.961.37
+FileExtensions:
+- html
+- htm
+- url
+- pdf
+- crx
+Protocols:
+- http
+- https
+MinimumOSVersion: 10.0.0.0
+InstallerType: msi
+Installers:
+- Architecture: x64
+  InstallerUrl: https://msedge.sf.dl.delivery.mp.microsoft.com/filestreamingservice/files/8edc9d93-2e60-4a6c-90b8-7b44d1829e61/MicrosoftEdgeBetaEnterpriseX64.msi
+  InstallerSha256: 5CC7EF546E31EF649944604A84CE0057F937E3865217C5B00C7B3B072F809D37
+  ProductCode: "{91FB278B-E80F-37BD-B822-B59DEAE0CAA6}"
+  Scope: machine
+  InstallerLocale: en-US
+- Architecture: x86
+  InstallerUrl: https://msedge.sf.dl.delivery.mp.microsoft.com/filestreamingservice/files/56a5f134-b47e-4196-a37b-5000b7a4409b/MicrosoftEdgeBetaEnterpriseX86.msi
+  InstallerSha256: 74221D5D93791A0B75531F8695E9B0790C5BD15A8AD75F6CD8CA9251342CED0F
+  ProductCode: "{A42ECA28-89A4-3E68-B72E-D8F0FD93DB48}"
+  Scope: machine
+  InstallerLocale: en-US
+- Architecture: arm64
+  InstallerUrl: https://msedge.sf.dl.delivery.mp.microsoft.com/filestreamingservice/files/ad42971a-3165-4046-969c-081d32626cbb/MicrosoftEdgeBetaEnterpriseARM64.msi
+  InstallerSha256: 6772EBC1F8255AA49756F760A4B5B8F8569D0A9E28D07EF80ABDFA8CF6283AA8
+  ProductCode: "{12134636-368E-34F8-8927-B23122634AA6}"
+  Scope: machine
+  InstallerLocale: en-US
```

Comment:
```suggestion
```
It looks like we need to start to move away from using `InstallerLocale` here because this prevents machines outside of the `en-US` locale from upgrading the application using the `winget upgrade` command.

See the comment that @denelon wrote at https://github.com/microsoft/winget-pkgs/issues/25347#issuecomment-903016232 & https://github.com/microsoft/winget-pkgs/issues/25347#issuecomment-903033816.